src/types.ts

```diff
@@ -0,0 +1,32 @@
+export type Cat = Readonly<{
+  id: string;
+  name: string;
+  url: string | null;
+}>;
+
+export type CatList = ReadonlyArray<Cat>;
+
+export type PageMetadata = Readonly<{
+  hasNextPage: boolean;
+}>;
+
+export type PagedResult = Readonly<{
+  results: CatList;
+  metadata: PageMetadata;
+}>;
+
+export type CatMetaData = Readonly<{
+  id: string;
+  name: string;
+  cTime: string;
+  mTime: string;
```

Comment:
```suggestion
  ctime: string;
  mtime: string;
```
I believe these fields are returned as `mtime` and `ctime` and not `m_time`/`c_time` so using the above names will not work.

Reply:
done